storio-common-annotations-processor/src/main/java/com/pushtorefresh/storio/common/annotations/processor/introspection/StorIOColumnMeta.java

```diff
@@ -41,6 +41,16 @@ public boolean isMethod() {
         return element.getKind() == ElementKind.METHOD;
     }
 
+    public String getRealElementName() {
+        if (elementName.startsWith("get") && Character.isUpperCase(elementName.charAt(3))) {
+            return elementName.substring(3).toLowerCase();
```

Comment:
It doesn't interfere with upper case symbols in the middle. I am just stripping out get or is part from getters. It could fail for unconventional names (starting from upper case e.g.) but I don't really know how to handle it.